package.json

```diff
@@ -12,7 +12,8 @@
     "classnames": "^2.3.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "^4.0.3"
+    "react-scripts": "^4.0.3",
+    "uuid": "^9.0.0"
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use `git add src/.` & `git commit -m 'your message'`